docs/content/Coding Milestones/PoC2023q1/commands.md

```diff
@@ -257,8 +257,24 @@ the ESPW.
 These commands administer a deployment of the central components ---
 the kcp server, PKI, and the central KubeStellar components --- in a
 Kubernetes cluster that will be referred to as "the hosting cluster".
-These are framed as "kubectl plugins" and thus need to be explicitly
-or implicitly given a kubeconfig file for the hosting cluster.
+These commands are framed as "kubectl plugins" and thus need to be
+explicitly or implicitly given a kubeconfig file for the hosting
+cluster.
+
+You need a Kubernetes cluster with an Ingress controller deployed and
+configured in a way that does _not_ terminate TLS connections (this
+abstinence is often called "TLS passthrough"). An OpenShift cluster
+would be one qualifying thing. Another would be an ordinary Kubernetes
+cluster with the [nginx Ingress
+controller](https://docs.nginx.com/nginx-ingress-controller/) deployed
+and configured appropriately. Please note that [special
```

Comment:
Should we add a pointer to a public guide/example of configuring TLS passthrough with nginx ?

Reply:
Yes, I did that in the extended example. I thought that the best place for an example.

Reply:
Since then I have removed duplication, leaving the main content here, and put the pointer here.